lib/screens/landing_screen.dart

```diff
@@ -21,120 +22,103 @@ class LandingPage extends StatelessWidget {
       elevation: 5,
     );
 
-    return Stack(
-      children: [
-        Container(
-          decoration: const BoxDecoration(
-            image: DecorationImage(
-              image: AssetImage('assets/backgrounds/Landing-Page.png'),
-              fit: BoxFit.fill,
-            ),
+    return Scaffold(
```

Comment:
Why is there a scaffold instead of a stack if you're only using the body widget?